blockstore/src/test/java/brooklyn/location/blockstore/AbstractVolumeManagerLiveTest.java

```diff
@@ -129,7 +136,9 @@ public void testCreateVolume() throws Exception {
                 .deviceSuffix(getDefaultDeviceSuffix())
                 .sizeInGb(getVolumeSize())
                 .tags(tags);
-        volume = volumeManager.createBlockDevice(jcloudsLocation, options);
+        currentJcloudshMachineLocation = createJcloudsMachine();
```

Comment:
This is only `testCreateVolume` and shouldn't require creating instance.
Is it vcloud-specific?